bynder_sdk/util.py

```diff
@@ -5,7 +5,9 @@
 }
 
 
-def api_endpoint_url(session, endpoint):
+def api_endpoint_url(session, endpoint, is_fs_endpoint=False):
```

Comment:
Feels somewhat cumbersome to have to pass `is_fs_endpoint` now everytime. And say in the future some new api gets added, we would need to add another arg. Maybe better to just check if it starts with `/v6/` or `/v7/`?

Or just prefixing the other routes with `/api` and always returning `'https://{}{}'` might be the most robust / future-proof solution.

Reply:
Sounds good!